src/components/BaseListGroupItemAssetPool.vue

```diff
@@ -15,26 +25,29 @@
             <small class="text-muted text-overflow-75">{{ membership.poolAddress }}</small>
         </div>
 
-        <div class="h3 mr-3 m-0">{{ membership.token.balance }} {{ membership.token.symbol }}</div>
+        <div v-if="membership.token" class="h3 mr-3 m-0">
+            {{ membership.token.balance }} {{ membership.token.symbol }}
```

Comment:
@GarfDev Please rebase your wallet branch and update api@main to verify if this still works as expected. I have changed this to `membership.token.poolBalance`.